S/SEAL/bundled/patches/fix_cache_init_cross-compilation_issue.patch

```diff
@@ -1,7 +1,7 @@
-diff --git before/SEAL-3.5.5/CMakeLists.txt after/SEAL-3.5.5/CMakeLists.txt
+diff --git before/SEAL-3.5.6/CMakeLists.txt after/SEAL-3.5.6/CMakeLists.txt
```

Comment:
Wouldn't it be better to refresh the patch to not have the top-level directory hard-coded? Something like
```
a/CMakeLists.txt
b/CMakeLists.txt
```

Reply:
Alas, you are right... I tried to fix it in d141fa0 but if it doesn't work, I'll probably have to rescind the PR and do it properly in a few days, since I do not have access to a proper machine for trying it inside a proper BinaryBuilder environment right now.

Reply:
Seems it worked, thanks!